plugins/Kaleidoscope-Qukeys/src/kaleidoscope/plugin/Qukeys.h

```diff
@@ -244,3 +244,10 @@ extern kaleidoscope::plugin::Qukeys Qukeys;
       qukey_defs};                                                  \
     Qukeys.configureQukeys(qk_table);                               \
   }
+
+/// This defines a `Key` object that will be handled by the Qukey plugin.
+/// The argument `n` is the index number of the `Qukey` in the array (starting
+/// at zero).
+constexpr kaleidoscope::Key QK(uint8_t n) {
+  return kaleidoscope::Key(kaleidoscope::ranges::QK_FIRST + n);
+}
```

Comment:
Mostly copy/paste from CharShift. I noticed that `LayerTapKey` and `ModTapKey` are defined within `kaleidoscope::plugin` above and that macros are used to expose those in a user friendly way. Any preference on either of those?